platform/local_country_file_utils.hpp

```diff
@@ -55,6 +55,7 @@ void CleanupMapsDirectory(int64_t latestVersion);
 // Tries to parse a version from a string of size not longer than 18
 // symbols and representing an unsigned decimal number. Leading zeroes
 // are allowed.
+// This function doesn't modify version param if s param can't be parsed.
```

Comment:
In this file params in comments are wrapped in || braces. So, could you please paraphrase the comment:
```c++
// This function doesn't modify |version| if |s| can't be parsed.
```
Also, please, reformat the whole comment - the short line above looks quite alone.